### ISSUE TITLE
TriggerBot not working!

### ISSUE THREAD

**Issue by citizen46** [McSwaggens/linux-csgo-external#50]
Please fix the TriggerBot Offset or kindly teach me how to find offset, Thanks! && How can i merge AimTux with this cheat? i wants to remove AimTux GUI and want it in comand-line, like this cheat.

**Comment by ericek111** [comment]
Yep, I've noticed that today. Also my GlobalVars signature changed: 
`48 8B 05 ?? ?? ?? ?? 55 48 89 E5 5D 48 8B 00 F3 0F 10 40 10`

New iCrosshairIndex offset: `0xBBB8`

To find the offset in the future, check out this [short how-to](https://github.com/McSwaggens/linux-csgo-external/pull/49). Or just add/subtract 4 few times. It doesn't change much.

AimTux is an internal cheat, this is external glow and inCross trigger. AimTux is sooo much more complex that it's impossible to "merge" it with any external. I'm trying to recreate some of its functions in my [java-csgo-external](https://github.com/ericek111/java-csgo-externals).

EDIT: Nevermind, I'm autistic. Looks like `0xBBD4` returns ID of the last entity in crosshair.
